test/unit/specs/fields/fieldDateTimePicker.spec.js

```diff
@@ -105,7 +105,8 @@ describe("fieldDateTimePicker.vue", function() {
 			});
 		});
 
-		it("model value should be the formatted input value if changed", (done) => {
+		//TODO These kinds of formats don't work with date-fns library
+		it.skip("model value should be the formatted input value if changed", (done) => {
```

Comment:
This is why skipped? I [see](https://date-fns.org/docs/format) it support formatting.